server/src/uuid/model/entity/mod.rs

```diff
@@ -990,6 +995,64 @@ impl Entity {
     }
 }
 
+impl Entity {
+    pub async fn deleted_entities<'a, E>(
+        payload: &deleted_entities_query::Payload,
+        executor: E,
+    ) -> Result<Vec<DeletedEntity>, sqlx::Error>
+    where
+        E: Executor<'a>,
+    {
+        let date_database: Option<DateTime>;
+        let date_ref = payload.after.as_ref();
+        match date_ref {
+            Some(date) => {
+                let date_chrono = chrono::DateTime::parse_from_rfc3339(&(date))
```

Comment:
on the other hand, it is not that good to parse it here, unwrapping an error could stop the application. Maybe the API could send a date as simple `number` , what do you think @kulla ?

Reply:
Eventuell operation::Error::BadRequest zurückgeben?

Reply:
Yes that is right